sympy/solvers/ode/tests/test_single.py

```diff
@@ -1751,6 +1764,35 @@ def _get_examples_ode_sol_2nd_2F1_hypergeometric():
     }
     }
 
+@_add_example_keys
+def _get_examples_ode_sol_2nd_nonlinear_autonomous_conserved():
+    return {
+        'hint': "2nd_nonlinear_autonomous_conserved",
+        'func': f(x),
+        'examples': {
+            '2nd_nonlinear_autonomous_conserved_01': {
+                'eq': f(x).diff(x, 2) + exp(f(x)) + log(f(x)),
+                'sol': [Eq(Integral(1/sqrt(C1 - 2*u*log(u) + 2*u - 2*exp(u)), (u, 1, f(x))), C2 + x), Eq(Integral(1/sqrt(C1 - 2*u*log(u) + 2*u - 2*exp(u)), (u, 1, f(x))), C2 - x)],
+                'simplify_flag': False,
```

Comment:
Some of the lines here are too long and should be formatted more nicely.

It is better not to fully indent these (like the other tests)

Reply:
I'll split it up into 2 lines.

Reply:
Do you want me to split the tests into lines like this test?
https://github.com/sympy/sympy/blob/4c20029861a0cdb8ab2356b6f57cc8f8d5b2b8b4/sympy/solvers/ode/tests/test_single.py#L1868-L1874
Is there any reason why the tests shouldn't be indented? It looks cleaner with the indent IMO.

Reply:
We want to stop the line length being too long. Adding more indentation means that more equations have to be wrapped across lines.

I'm guessing that you are asking about this because your editor is trying to force the indentation or something. We could use standard indentation rules like:
```python
    examples = {}
    example_set = {
        'hint': "2nd_nonlinear_autonomous_conserved",
        'func': f(x),
        'examples': examples,
    }
    examples['2nd_nonlinear_autonomous_conserved_01'] = {
        'eq': f(x).diff(x, 2) + exp(f(x)) + log(f(x)),
        'sol': [
            Eq(Integral(1/sqrt(C1 - 2*u*log(u) + 2*u - 2*exp(u)), (u, 1, f(x))), C2 + x),
            Eq(Integral(1/sqrt(C1 - 2*u*log(u) + 2*u - 2*exp(u)), (u, 1, f(x))), C2 - x),
        ],
        'simplify_flag': False,
    }
    return example_set
```
I think that's also fine. However we should be consistent across the different tests in the same file so unless we are planning to change *all* the examples to this format then I think the tests here should match what is done in the rest of the file.

@Mohitbalwani26 what do you think?

Reply:
In my opinion, Only the example `separable_reduced_11` is too long rest all looks better to me as they are now. If we try to use your approach, Indention would be almost same as most of the examples look like this.
```python3
def _get_examples_ode_sol_separable_reduced():
    df = f(x).diff(x)
    return {
            'hint': "separable_reduced",
            'func': f(x),
            'examples':{
    'separable_reduced_01': {
        'eq': x* df  + f(x)* (1 / (x**2*f(x) - 1)),
        'sol': [Eq(log(x**2*f(x))/3 + log(x**2*f(x) - Rational(3, 2))/6, C1 + log(x))],
        'simplify_flag': False,
        'XFAIL': ['lie_group'], #It hangs.
    },
    }
    }
```

Reply:
Yes, from what I observed, there was only 1 example that's too long. I'll look throughout the file change lines that are not too long into this format.